_posts/2020-05-01-newsletter.md

```diff
@@ -0,0 +1,165 @@
+  - `nvim_create_buf`: create a new buffer (**@bredl**, [#9272](https://github.com/neovim/neovim/pull/9272))
+  - `nvim_get_context`/`nvim_load_context`:  explore and restore the editor's state (**@justinmk**, [#10619](https://github.com/neovim/neovim/pull/10619).
+  - `nvim_input_mouse`: perform mouse actions (**@bfredl**, [#9429](https://github.com/neovim/neovim/pull9429)).
+  - `nvim_open_win`: creating floating external windows 
+  - `nvim_set_keymap`: sets a global mapping for a mode (**@yilin-yang**, [#9224](https://github.com/neovim/neovim/pull/9224)).
+  - `nvim_win_close`: close a window (**@bfredl**, [#9667](https://github.com/neovim/neovim/pull/9667)).
+  - `nvim_win_set_buf`: sets the current buffer of a window (**@justinmk**, [#9100](https://github.com/neovim/neovim/pull/9100))
+  - `nvim_win_set_config`: configure the layout of a window (**@mhinz**, [#9626](https://github.com/neovim/neovim/pull/9626)).
+
+- Changed API functions: 
+  - `nvim_buf_lines_event` will now send events for terminal buffers (**@justinmk**, [#8616](https://github.com/neovim/neovim/pull/8616)).
+
+- Neovim now always waits for a UI when usied with --embed, unless --headless is also supplied (**@bfredl**, [#9024](https://github.com/neovim/neovim/pull/9024)).
+
+- Neovim's libuv loop has been exposed to lua as `vim.loop` (**@zhaozg**, **@andreypopp**, [#10123](https://github.com/neovim/neovim/pull/10123)).
+
+### The near future: Upcoming
+
+
+Around Neovim
+-------------
+
+
+## Related projects
+
+The wiki page of [related
+projects](https://github.com/neovim/neovim/wiki/Related-projects) is an evergrowing list of projects that somehow make use of Neovim's extended capabilities. Here's a gist of additions since the last newsletter: 
+
+### Plugins
+
+The following plugins have found their way into our [wiki](https://github.com/neovim/neovim/wiki):
+
+- [coc.nvim](https://github.com/neoclide/coc.nvim): Language Server Protocol client that aims to replicate VSCode's functionnalities
+- [markdown-preview.nvim](https://github.com/iamcco/markdown-preview.nvim): Provides preview of your markdown files in your browser, with synchronized scrolling.
+- [nvim-gdb](https://github.com/sakhnik/nvim-gdb): Neovim thin wrapper for GDB, LLDB and PDB
+- [nero.nvim](https://github.com/Vigemus/nero.nvim): A REPL for Neovim
+- [neogdb.vim](https://github.com/huawenyu/neogdb.vim): Vim GDB front-end for neovim
+- [nvim-luadev](https://github.com/bfredl/nvim-luadev): REPL for developing lua plugins
+- [Conjure](https://github.com/Olical/conjure): Clojure and ClojureScript tooling for Neovim over a socket
+
+
+### API Clients
+
+Plenty of new API clients:
+
+- Java: [neovim-java](https://github.com/esensar/neovim-java)
+- D: [viniarck/nvimhost-d](https://github.com/viniarck/nvimhost-d)
+- Scala: [viniarck/nvimhost-scala](https://github.com/viniarck/nvimhost-scala)
```

Comment:
We may add https://github.com/smolck/dart-nvim-api.

Reply:
This list currently only references bindings that came out between v0.3.0 and v0.4.0. It's true that if the list is extended to go from v0.3.0 to v0.4.0, dart should be added.